### ISSUE TITLE
Media with custom fields can't be saved

### ISSUE THREAD

**Issue by erixtekila** [pods-framework/pods#7158]
### Description

On the last version 2.9.1, adding a field to the Media post_type works.
But using the customs fields in the media UI, can’t be saved with the publish button.

In my case, I add other pod relationship.
The edition is not saved afterward.

### Version

2.9.1

### Testing Instructions

1. Create a pod by extending Media
2. Add a relation, for exemple another CPT pod.
3. Go edit an image in the dedicated interface
4. Add a new relation
5. Save
6. Quit the page
7. Return to it, nothing has been saved.

### Pods Package

```json
{
    "@meta": {
        "version": "2.9.19",
        "build": 1694163437
    },
    "settings": {
        "types_only": "0",
        "watch_changed_fields": "0",
        "metadata_integration": "1",
        "metadata_override_get": "0",
        "media_modal_fields": "0",
        "session_auto_start": "0",
        "wisdom_opt_out": "1"
    },
    "pods": [
        {
            "name": "media",
            "id": 127,
            "label": "Media",
            "description": "",
            "type": "media",
            "storage": "meta",
            "object": "media",
            "_migrated_28": "1",
            "required": "0",
            "required_help_boolean": "0",
            "write_all": "media",
            "rest_api_field_mode": "value",
            "rest_enable": "1",
            "groups": [
                {
                    "name": "lien",
                    "id": 128,
                    "label": "Lien associ\u00e9 \u00e0 l'image",
                    "description": "",
                    "weight": 0,
                    "roles_allowed": "administrator",
                    "fields": [
                        {
                            "name": "projet",
                            "id": 129,
                            "label": "Projet",
                            "description": "Lien vers le projet associ\u00e9",
                            "weight": 0,
                            "type": "pick",
                            "pick_object": "post_type",
                            "pick_val": "projet",
                            "sister_id": "--\u00a0S\u00e9lectionnez\u00a0--",
                            "pick_table": "--\u00a0S\u00e9lectionnez\u00a0--",
                            "required": "0",
                            "required_help_boolean": "0",
                            "pick_format_type": "single",
                            "pick_format_single": "dropdown",
                            "pick_format_single_help": "0",
                            "pick_format_multi": "list",
                            "pick_format_multi_help": "0",
                            "pick_display_format_multi": "default",
                            "pick_display_format_separator": ", ",
                            "pick_allow_add_new": "1",
                            "pick_taggable": "0",
                            "pick_show_icon": "1",
                            "pick_show_edit_link": "1",
                            "pick_show_view_link": "1",
                            "pick_limit": "0",
                            "pick_user_role": "Administrator",
                            "pick_post_status": "publish",
                            "pick_post_author": "0",
                            "repeatable": "0",
                            "repeatable_format": "default",
                            "roles_allowed": "administrator",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "1"
                        },
                        {
                            "name": "artisan",
                            "id": 130,
                            "label": "Artisan",
                            "description": "Lien vers un artisan",
                            "weight": 1,
                            "type": "pick",
                            "pick_object": "post_type",
                            "pick_val": "artisan",
                            "sister_id": "--\u00a0S\u00e9lectionnez\u00a0--",
                            "pick_table": "--\u00a0S\u00e9lectionnez\u00a0--",
                            "required": "0",
                            "required_help_boolean": "0",
                            "pick_format_type": "single",
                            "pick_format_single": "dropdown",
                            "pick_format_single_help": "0",
                            "pick_format_multi": "list",
                            "pick_format_multi_help": "0",
                            "pick_display_format_multi": "default",
                            "pick_display_format_separator": ", ",
                            "pick_allow_add_new": "1",
                            "pick_taggable": "0",
                            "pick_show_icon": "1",
                            "pick_show_edit_link": "1",
                            "pick_show_view_link": "1",
                            "pick_limit": "0",
                            "pick_user_role": "Administrator",
                            "pick_post_status": "publish",
                            "pick_post_author": "0",
                            "repeatable": "0",
                            "repeatable_format": "default",
                            "roles_allowed": "administrator",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "1"
                        }
                    ]
                }
            ]
        }
    ]
}
```

**Comment by sc0ttkclark** [comment]
I've got a fix for this in Pods 3.0, just made it before release!

**Comment by sc0ttkclark** [comment]
Prepping everything as we speak, hopefully by noon today (UTC-5)
